hw2/aes_power_analysis.py

```diff
@@ -172,6 +188,28 @@ def power_analysis_attack(key_len, plaintexts, hamming_distances, verbose=False)
     return recover_full_key(key_len, k0_list, k1_list)
 
 
+def check_test_vectors_192():
+    num_traces = 10
+    plaintext_seed = 0
+
+    key = '000102030405060708090a0b0c0d0e0f101112131415161718191a1b1c1d1e1f'

```

Comment:
Isn't this 256 bit key? Need to shorten it.

Reply:
Changed to `000102030405060708090a0b0c0d0e0f1011121314151617`